### PR TITLE
add target limit

### PR DESCRIPTION
**Pull request by brunoparma88**
create target limit for user

### CODE REVIEW

File: app/models/target.rb [hunk 1]
```diff
@@ -22,4 +22,11 @@ class Target < ApplicationRecord
   belongs_to :topic
 
   validates :longitude, :latitude, :radius, :title, :user, :topic, presence: true
+  validate :mount_of_targets, on: :create
+
+  private
+
+  def mount_of_targets
+    errors.add('You reached the limit of targets') if user.targets.count > 10
```
Comment (jpascual1994) [inline app/models/target.rb]: Maybe we can use I18n for the error message
https://guides.rubyonrails.org/i18n.html

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,33 +1,3 @@
...
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
-
 en:
-  hello: "Hello world"
+  error:
+    target_limit: "'You reached the limit of targets'."
```
Comment (jpascual1994) [inline config/locales/en.yml]: Why the `"'`?
```suggestion
    target_limit: 'You reached the limit of targets.'
```

File: app/models/target.rb [hunk 3]
```diff
@@ -22,4 +22,11 @@ class Target < ApplicationRecord
   belongs_to :topic
 
   validates :longitude, :latitude, :radius, :title, :user, :topic, presence: true
+  validate :mount_of_targets, on: :create
+
+  private
+
+  def mount_of_targets
+    errors.add(I18n.t('error.target_limit')) if user.targets.count > 10
```
Comment (martinjaimem) [inline app/models/target.rb]: we could declare 10 as a constant

File: spec/models/target_spec.rb [hunk 4]
```diff
@@ -1,7 +1,27 @@
...
+# Indexes
+#
+#  index_targets_on_topic_id  (topic_id)
+#  index_targets_on_user_id   (user_id)
+#
 require 'rails_helper'
 
 describe Target do
   describe 'Associations' do
+    subject { create :target, user: create(:user), topic: create(:topic) }
```
Comment (martinjaimem) [inline spec/models/target_spec.rb:24]: we should be consistent with the parenthesis... `create(`

**Review by martinjaimem — APPROVED** [review]

**Review by jpascual1994 — APPROVED** [review]

**Pull request merged**